beacon-chain/rpc/eth/validator/validator.go

```diff
@@ -494,8 +494,34 @@ func (vs *Server) ProduceSyncCommitteeContribution(
 	panic("implement me")
 }
 
-func (vs *Server) SubmitContributionAndProofs(ctx context.Context, request *ethpbv2.SubmitContributionAndProofsRequest) (*empty.Empty, error) {
-	panic("implement me")
+// SubmitContributionAndProofs publishes multiple signed sync committee contribution and proofs.
+func (vs *Server) SubmitContributionAndProofs(ctx context.Context, req *ethpbv2.SubmitContributionAndProofsRequest) (*empty.Empty, error) {
+	ctx, span := trace.StartSpan(ctx, "validator.SubmitContributionAndProofs")
+	defer span.End()
+
+	for _, item := range req.Data {
+		v1alpha1Req := &ethpbalpha.SignedContributionAndProof{
+			Message: &ethpbalpha.ContributionAndProof{
+				AggregatorIndex: item.Message.AggregatorIndex,
+				Contribution: &ethpbalpha.SyncCommitteeContribution{
+					Slot:              item.Message.Contribution.Slot,
+					BlockRoot:         item.Message.Contribution.BeaconBlockRoot,
+					SubcommitteeIndex: item.Message.Contribution.SubcommitteeIndex,
+					AggregationBits:   item.Message.Contribution.AggregationBits,
+					Signature:         item.Message.Contribution.Signature,
+				},
+				SelectionProof: item.Message.SelectionProof,
+			},
+			Signature: item.Signature,
+		}
+		_, err := vs.V1Alpha1Server.SubmitSignedContributionAndProof(ctx, v1alpha1Req)
```

Comment:
Maybe there should be a batch endpoint in v1alpha1 for submitting multiple instead of issuing N requests

Reply:
Something to keep in mind. If we feel like it will be a problem, we can open a tracking issue to do this